content/support-matrix/delfin/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Delfin
+menuTitle: Delfin
+description: "Support matrix data for drivers of Delfin project"
+weight: 10
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"] 
+---
+
+Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform.
```

Comment:
This content needs to be modified . 
Please take below content and modify if required..

Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform. Delfin has standardized common API services.
Support matrix of each storage driver are grouped to their vendor name.
Each driver support matrix include below topics.
1.	Model and management software versions supported:-  List of storage models and required management software versions.
2.	Access information to register storage: Table illustrates what all access protocol supported and parameters required to register the storage device.
3.	Alert management information : Type of alert sourcing and clearing supported

4.	Performance registration information:  Table illustrates the resources and their default interval supported for performance collection.
5.	Resource data matrix:  Table illustrates the resources supported anlog with their available attributes.
6.	Alert data matrix : Table illustrates fields available  in the alert data 
7.	Performance metric data: Table illustrates performance metrics supported for each resource.